README.md

```diff
@@ -85,10 +85,12 @@ FLAGS:
     -V, --version                Prints version information
 
 OPTIONS:
+        --branch <branch>         Specify a git branch to download the crate from
```

Comment:
This change is probably an issue with the conflict resolution. Don't think you need this.

Reply:
I just ran `cargo run --bin cargo-add add -h` and committed the result. I can split out the unrelated changes into their own PR if you would prefer.

Reply:
Looks like they forgot to update the README. So, I guess, it is okay.